debian/grub-acrn.README.Debian

```diff
@@ -0,0 +1,26 @@
+
+ACRN specific GRUB configuration items
+--------------------------------------
+
+The following variables can be set and exported in /etc/default/grub.d/acrn.cfg
+to control GRUB boot menu generation for ACRN:
+
+GRUB_CMDLINE_ACRN: command line parameter for the ACRN hypervisor. This will
+                   be merged with parameters from ACRN configuration. E.g.
+                   add a required uart parameter, see
+                   https://projectacrn.github.io/latest/user-guides/hv-parameters.html
+
+GRUB_CMDLINE_LINUX and GRUB_CMDLINE_LINUX_DEFAULT (in /etc/default/grub) can be
+used as usual and apply to the ACRN Service VM only. They are appended to the
+settings provided by ACRN configuration.
+
+ACRN VM specific variables (x means the VM id in the scenario configuration):
+
+GRUB_ACRN_KERNEL_CMDLINE_VMx: kernel command line parameter for VMx
+Attention: Using GRUB_ACRN_KERNEL_CMDLINE_VMx for Service VM will *OVERRIDE*
+           the settings provided by ACRN configuration and is *NOT*
+           recommended! So, be careful!
+The following parameters are valid for pre-launched VMs only:
+GRUB_ACRN_KERNEL_VMx: path of VMx kernel
+GRUB_ACRN_RAMDISK_VMx: path of VMx ramdisk
+GRUB_ACRN_ACPI_VMx: path of VMx ACPI data: defaults to /boot/ACPI_VMx.bin
```

Comment:
The vACPI binaries are generated at build time and installed under /boot/ by our Debian packages. Does any user really need `GRUB_ACRN_ACPI_VMx` in any case?

Reply:
Right now you don't need it (and you don't have to, sincxe ther is the default location). I just thought kernel, initrd and acpi data are the three possible blobs we nee to load, so to be on the flexible side I just added all of them. Nevertheless, if we use the module tags as identifier as mentioned above, we do not even have to distinguish the three types of blobs since we will identify them via their module tag names. So, I propose to use ACRN_MOD_<tag name> for all of them, and using /boot/ACPI_VMx.bin as default blob, if not set explicitly in the config. What do you think?

Reply:
`ACRN_MOD_` should read `GRUB_ACRN_MOD_<mod tag name>`. Sorry for that!

Reply:
> Right now you don't need it (and you don't have to, sincxe ther is the default location). I just thought kernel, initrd and acpi data are the three possible blobs we nee to load, so to be on the flexible side I just added all of them. Nevertheless, if we use the module tags as identifier as mentioned above, we do not even have to distinguish the three types of blobs since we will identify them via their module tag names. So, I propose to use ACRN_MOD_ for all of them, and using /boot/ACPI_VMx.bin as default blob, if not set explicitly in the config. What do you think?

Looks good to me too :)